templates/mask_generation.ipynb

```diff
@@ -0,0 +1,298 @@
+{
```

Comment:
<p>Okay, I wasn't clear about this. The image here won't already be generated: it'll be a feature in the cell table. For example, we might want to color each cell by the area of the cell, instead of the cell type. So this should take in a column of numeric values, rather than categorical values, and use that to color each cell in the image. The output should look identical to the cohort_cluster_plot, but instead of categorical colors it'll be a single colormap across the dynamic range of the feature</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/1052/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>